lib/src/summarize.dart

```diff
@@ -4,12 +4,51 @@
 
 library dart_pad.summarize;
 
+import '../services/dartservices.dart';
 /// Instances of this class take string input of dart code and output a text 
 /// description ofthe code's size, packages, and other useful information
 class Summarizer {
   
-  String summarize(String input) {
-    return "foo";
+  String summarize(SourceRequest input, AnalysisResults result) {
+    if (input == null || result == null) {
+      return "Summarizer has broken!.";
+    }
+    String summary = '<pre><code><b>Summary (Under Development)</b><p/>';
```

Comment:
Let's return the summary in a plain text format (i.e. not html). Markdown might be interesting, if we used it sparingly.

The summary should really read like 1-3 hand-written sentences describing the gist.